Applications/SlicerApp/Testing/Python/SlicerBoundsTest.py

```diff
@@ -71,8 +71,8 @@ def test_Volume(self):
     """ Test the GetRASBounds & GetBounds method on a volume.
     """
     #self.delayDisplay("Starting test_Volume")
-    from SampleData import SampleDataLogic
-    volumeNode = SampleDataLogic().downloadAbdominalCTVolume()
+    import SampleData
+    volumeNode = SampleData.downloadSample('CTA abdomen\n(Panoramix)')[0]
```

Comment:
@lassoan  I don't link this name including a newline ... I think we should have a SampleName and SampleDescription.